oc_logging/test/test.py

```
@@ -21,6 +21,16 @@ def _raise_zero_division():
         log.error("failed", exc_info=True, key="value")
 
 
+def _log_exception_object():
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `_log_exception_object`

Add return type annotation: `None`

(ANN202)